src/types/algoliasearch.ts

```diff
@@ -19,11 +19,12 @@ import {
   SearchResponse as SearchResponseV4,
   // no comma, TS is particular about which nodes expose comments
   // eslint-disable-next-line prettier/prettier
-  SearchForFacetValuesResponse as SearchForFacetValuesResponseV4
+  SearchForFacetValuesResponse as SearchForFacetValuesResponseV4,
   /** @ts-ignore */
   // @ts-ignore
   // eslint-disable-next-line import/no-unresolved
 } from '@algolia/client-search';
+export { FindAnswersResponse } from '@algolia/client-search';
```

Comment:
this one can be consolidated with the other imports

Reply:
https://github.com/algolia/instantsearch.js/pull/4581/commits/3c3e592a75e7ed7e8bbd7ce1ac1e17535addf013